ua-snap/mv-aicc-fire-shim
# Refresh 2020

Updates URLs to working endpoints.

To test this,

```
rm -rf node_modules
npm install
npm test   # should not report errors
npm start
```

then open `localhost:3000/fires.geojson` in your web browser.  You should receive a little blorp of data.  If you don't and there are errors about verifying certificates in the standard out for the app (MacOS Catalina) then `export NODE_DEBUG=true` and try again.

Next, open `localhost:3000/viirs.geojson` in a web browser.  You should only see lat/lon pairs in there, not fields like CONFIDENCE or OBSERVEDTIME.

Finally, in the diff, confirm that a large section of code which had managed the fire tally computations has been removed.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ua-snap/mv-aicc-fire-shim/36)
